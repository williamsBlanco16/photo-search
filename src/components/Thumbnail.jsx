import React from 'react'
import PropTypes from 'prop-types'

const Thumbnail = ({src})=> {

  return(
    <img src={src}
      style = {{
        width:'300px',
        height:'300px'
      }} />
  );
}

Thumbnail.propTypes = {
  src: PropTypes.string.isRequired
}

export default Thumbnail;