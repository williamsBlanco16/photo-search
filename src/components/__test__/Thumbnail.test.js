import React from 'react'
import renderer from 'react-test-renderer'

import Thumbnail from '../Thumbnail'

test('Thumbnail renderes an img tag with the correct src', () => {
  const component = renderer.create(
    <Thumbnail src = 'https://test.com/test.jpg'/>
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
