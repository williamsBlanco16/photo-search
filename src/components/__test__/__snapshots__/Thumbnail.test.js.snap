// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail renderes an img tag with the correct src 1`] = `
<img
  src="https://test.com/test.jpg"
  style={
    Object {
      "height": "300px",
      "width": "300px",
    }
  }
/>
`;
